src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,126 @@
+// ----- BASE ELEMENTS ---------------
+
+.eyebrow {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
```

Comment:
Please update to use HelixUI color vars:

https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/_vars.scss

Reply:
Changes have been done, Please review